lib/mayaUsd/nodes/layerManager.cpp

```diff
@@ -742,20 +767,11 @@ SaveStageToMayaResult saveStageToMayaFile(
 
     const MFnDependencyNode depNodeFn(proxyNode);
     MayaUsdProxyShapeBase*  pShape = static_cast<MayaUsdProxyShapeBase*>(depNodeFn.userNode());
+    if (!pShape)
+        return result;
 
-    SdfLayerHandleVector allLayers = stage->GetLayerStack(true);
```

Comment:
Can we just replace this with `auto allLayers = UsdUfe::getAllSublayerRefs(layer, includeTopLayer);` ?